source/documentation/10-switching-to-production.md

```diff
@@ -91,7 +89,7 @@ Go to _Configuration > Technical information_
 1. On the _Technical information_ page, click the _Data and origin verification_ tab.
 1. Scroll to the _Checks for e-Commerce & Alias Gateway_ section.
 1. Leave the _URL of the merchant page containing the payment form that will call the page: orderstandard.asp_ field blank.
-Enter a strong SHA-IN passphrase; this will be used when setting up the GOV.UK account credentials.
+1. Enter a strong plain text string SHA-IN passphrase; this will be used when setting up the GOV.UK account credentials.
```

Comment:
Looks good. Minor thing: is it `plain text string`, or `plain-text string`? You maybe write `plain text`, but if you use it as an apposition for `string`, you may need the hyphen?

Reply:
@daniele-occhipinti good point. I've checked and it seems to be written as "plaintext" when it is used for human-readable data that is viewable without any sort of decryption required. I'll amend.

@roryjhs @alexbishop1 any further comments on this?